content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -1,95 +1,93 @@
 
-To create a BoM from the Manufacturing module, go to :menuselection:`Products --> Bills of Materials`. 
-If you click **Create**, the first thing you'll have to do is to specify the final product. 
-For an existing product, simply select it from the list, or create a new one on the spot. Note 
-that if you go through the product form to create your BOM, the product will already be set for you.
+The simplest :abbr:`BoM (Bill of Materials)` setup is one without operations or instructions, only
+components. In this case, the production is solely managed using *Manufacturing Orders*.
 
-For a standard Bill of Material, keep the default *BoM* type, which is *Manufacture this Product*. 
-Now specify the various components that make up the production of your final product and their 
-respective quantities. You can create components as you go, on the BOM, or create products beforehand,
-by going to the :menuselection:`Top Menu --> Products --> Create`, and add them later on. 
+To create a :abbr:`BoM (Bill of Materials)` from the :guilabel:`Manufacturing` module, go to
+:menuselection:`Products --> Bills of Materials`. Then, click :guilabel:`Create`. Next, specify the
+:guilabel:`Product`. Note that a :abbr:`BoM (Bill of Materials)` can also be created directly from
+the product form. In that case, the :guilabel:`Product` field is pre-filled.
 
+For a standard :abbr:`BoM (Bill of Materials)`, set the :guilabel:`BoM Type` to
+:guilabel:`Manufacture this Product`. Then, click :guilabel:`Add a Line` to specify the various
+components that make up the production of the final product and their respective quantities. New
+components can be created quickly through the :abbr:`BoM (Bill of Materials)`, or can be created
+beforehand in :menuselection:`Manufacturing --> Products --> Products --> Create`. Finally, click
+:guilabel:`Save` to finish creating the :abbr:`BoM (Bill of Materials)`.
 
-.. image:: bill_configuration/bom_1.png
-    :align: center
+.. image:: bill_configuration/bom-form.png
+   :align: center
+   :alt: Set up a Bill of Materials.
 
-.. warning::
-   The destination location should **not** be a scrap location. A scrap location is where you put
-   products that you don't need.
+Specify a Bill of Materials (BoM) for a Product Variant
+-------------------------------------------------------
 
-Using the same BoM to describe Variants
----------------------------------------
+:abbr:`BoMs (Bills of Materials)` can also be assigned to specific *Product Variants*, with two
+setup options available to choose from.
 
-As suggested above, you can use *BOMs* for specific *Product Variants*. 
-Once the various attributes have been configured on the product form, there 
-are two ways to configure the appropriate BoM for the respective product combination. 
+.. :note:: In order to assign :abbr:`BoMs (Bills of Materials)` to product variants, the product's
+   variant attributes must already be configured on the product form.
```

Comment:
Nitpick DR will bring up:
When formatting a tag for note, tip, etc., actually break the line after the tag's closing colon `:` and align the content underneath it so it lines up with the opening colon `:`, like so:
```suggestion
.. :note:: 
   In order to assign :abbr:`BoMs (Bills of Materials)` to product variants, the product's variant
   attributes must already be configured on the product form.
```